<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_form</class>
 <widget class="QDialog" name="edit_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Change the attribute if you think it has the wrong value.  To change any point of the polyline, click the left mouse button and drag the point to its new position.  To change the point's z-value, double-click the cell and type in the new value.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Road name:</string>
       </property>
       <property name="buddy">
        <cstring>name_combo_box</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="name_combo_box"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="type_label">
       <property name="text">
        <string>Road type:</string>
       </property>
       <property name="buddy">
        <cstring>type_combo_box</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="type_combo_box"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Polyline:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QTreeWidget" name="polyline_tree_widget">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>#</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>x</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>z</string>
        </property>
        <property name="toolTip">
         <string>Double click on this cell and type in the new Z value.</string>
        </property>
        <property name="statusTip">
         <string>Double click on this cell and type in the new Z value.</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="warning_label">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>edit_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>edit_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
